services/db/__tests__/GitHubService.spec.js

```diff
@@ -43,13 +43,13 @@ describe("Github Service", () => {
   })
 
   describe("getFilePath", () => {
-    it("Retrieves the right unlinked page file path", async () => {
+    it("Retrieves the right unlinked page file path (1)", async () => {
       expect(service.getFilePath({ siteName, fileName })).toEqual(
         `${siteName}/contents/${fileName}`
       )
     })
 
-    it("Retrieves the right unlinked page file path", async () => {
+    it("Retrieves the right unlinked page file path (2)", async () => {
```

Comment:
Retrieves the right collection page file path - the test below this should be for subcollection